core/server/master/src/main/java/alluxio/master/block/meta/MasterWorkerInfo.java

```diff
@@ -157,7 +157,7 @@ public MasterWorkerInfo(long id, WorkerNetAddress address) {
     }
 
     // Set the new block information.
-    mBlocks = new HashSet<>(blocks);
+    mBlocks = blocks;
```

Comment:
The `blocks` variable is a newly copied `HashSet` already, passed to this method. There's no benefit copying again.